BFF/v3/docs/config.toml

```diff
@@ -11,9 +11,8 @@ home = [ "HTML", "RSS", "JSON"]
 
 [params]
 editURL = "https://github.com/DuendeSoftware/docs.duendesoftware.com/edit/main/BFF/v3/docs/content/"
-qs_base = "https://github.com/DuendeSoftware/Samples/tree/main/IdentityServer/v7/Quickstarts"
-samples_base = "https://github.com/DuendeSoftware/Samples/tree/main/IdentityServer/v7"
-identityServerBaseURL = "https://docs.duendesoftware.com/bff/v3"
+samples_base = "https://github.com/DuendeSoftware/Samples/tree/main/Bff/v3"
+bffBaseUrl = "https://docs.duendesoftware.com/bff/v3"
```

Comment:
Did you try this specific change? I believe this is used in several partial/includes so may want to double check.

Reply:
not used afaik.